D20-160034.depth
Done :)
Traceback (most recent call last):
  File "make_plots.py", line 36, in <module>
    plot_depth.plot_depth("QC_depth/D20-160027.depth", "plots/" + id + "_depth_normalized.png", "Normalized Depth: Sample " + id, genome_size, True)
  File "/projectnb2/hfsp/Challenge21/helen/marine-library-resequencing/QC_depth/plot_depth.py", line 57, in plot_depth
    data = parse_depth(depth_report, genome_size)
  File "/projectnb2/hfsp/Challenge21/helen/marine-library-resequencing/QC_depth/plot_depth.py", line 28, in parse_depth
    with open(depth_input) as depth_object:
FileNotFoundError: [Errno 2] No such file or directory: 'QC_depth/D20-160027.depth'
